cpp-ch/local-engine/Storages/ch_parquet/OptimizedArrowColumnToCHColumn.h

```diff
@@ -4,7 +4,6 @@
 #include <Common/Config.h>
 
 #if USE_LOCAL_FORMATS
-// clang-format off
 #if USE_ARROW || USE_ORC || USE_PARQUET
```

Comment:
remove the `clang-format off` can work now?

Reply:
it does not influence building .